src/vinyldns/client.py

```diff
@@ -645,13 +645,14 @@ def approve_batch_change(self, batch_change_id, review=None):
 
         return BatchChange.from_dict(data)
 
-    def reject_batch_change(self, batch_change_id, review_comment=None):
+    def reject_batch_change(self, batch_change_id, review=None):
         """
         Reject a batch change
 
         :return: the content of the response
         """
-        url = urljoin(self.index_url, u'/zones/batchrecordchanges/{0}/reject'.format(batch_change_id))
+        url = urljoin(self.index_url, u'/zones/batchrecordchanges/{0}/reject'.format(batch_change_id),
```

Comment:
We also need optional `allowManualReview` parameter on the `create_batch_change` method.